### ISSUE TITLE
Decouple network access from member creation

### ISSUE THREAD

**Issue by Lodifice** [agdsn/pycroft#242]
Member creation shall be possible without giving a MAC address and a date of birth.

It shall be possible to enter the date from the services form (MAC address, date of birth) at any point. There needs to be a check that both fields are filled. Network access is best modelled as a separate property. It would be nice to have a button for revoking network access, this should involve removing the property from the user and deleting their MAC address and date of birth.

Solving this issue would enable work on https://github.com/agdsn/sipa/issues/385.

**Comment by toothstone** [comment]
Wasn't this resolved/implemented in https://github.com/agdsn/pycroft/commit/517338ef6c44ba5fe91e444a8bf8870f98a3aacb? Afaik we can close this issue.

**Comment by toothstone** [comment]
Or do we really need the 'revoke network access' button?

**Comment by ibot3** [comment]
There is a button to unblock a user since some time. There is a new "services only" group, but the member group was not split.
Closing this.
